examples/notebooks/positive_Helmholtz_3D_VTK.ipynb

```diff
@@ -0,0 +1,274 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "2443e734",
+   "metadata": {},
+   "source": [
+    "# 3D example with VTK and ParaView \n",
+    "\n",
+    "In this notebook we show how to solve a 3D problem in parallel, save the solution in HDF5 format and export it to VTK. \n",
+    "\n",
+    "We solve the positive Helmholtz equation in 3D with a callable function for the source. In practice, the source could be the interpolation of some experimental data. Besides, we impose periodic boundary conditions everywhere in the boundary.\n",
```

Comment:
I have two questions here: 
- why calling this a "Helmholtz equation" since it's vector-valued ?
- why using a positive value for mu, and not a negative ?
With a standard time-harmonic Maxwell problem the physical interpretation of this example would be clearer.

Reply:
fixed in https://github.com/pyccel/psydac/pull/543/commits/71dcf108804b7057f9f3b8eab65d55c266a39f05